src/ODINN.jl

```diff
@@ -31,6 +31,12 @@ using Downloads
 using TimerOutputs
 using GeoStats
 using ImageFiltering
+using EnzymeCore
+
+# This is equivalent to `@ignore_derivatives`
+EnzymeCore.EnzymeRules.inactive(::typeof(Huginn.define_callback_steps), args...) = nothing
+EnzymeCore.EnzymeRules.inactive(::typeof(Muninn.MB_timestep!), args...) = nothing
```

Comment:
@wsmoses this is the function that was inactivated but needed to be commented out due to hitting the PyCall thing.

Reply:
I suppose a long shot, but any chance you can make an MWE and post in the issue....or at least a version I can run to see what's going on?